evm/vm/forks/frontier/__init__.py

```diff
@@ -118,7 +118,7 @@ def _execute_frontier_transaction(vm, transaction):
             # The address of the newly created contract has *somehow* collided
             # with an existing contract address.
             computation = Computation(vm, message)
-            computation.error = ContractCreationCollision(
+            computation._error = ContractCreationCollision(
```

Comment:
explicit external setting of `_error` occurs in these fork `__init__.py` files. Violates the privateness of `_error`. Thoughts @pipermerriam?

Reply:
🏈   <-- punt on it and mark it with a `# TODO` comment.